### ISSUE TITLE
Outputs are imprecise

### ISSUE THREAD

**Issue by razorsedge** [cloudposse/terraform-aws-sns-topic#25]
## Describe the Bug
The output `sns_topic` should be the topic name and not a map of all attributes returned by `aws_sns_topic`.
```terraform
output "sns_topic" {
  value       = aws_sns_topic.this #<--
}
```
The same applies to `aws_sns_topic_subscriptions`.

## Expected Behavior
`sns_topic` should return only the topic.  Perhaps try `value       = aws_sns_topic.this.name`?

**Comment by nitrocode** [comment]
Why not simply use `module.this.sns_topic.name` instead of having to create an output for each resource output?

**Comment by razorsedge** [comment]
Because it conflicts with the output documentation?  Because it requires one to be an expert in `aws_sns_topic` outputs?  Because it is unexpected compared to all other CloudPosse modules?

**Comment by nitrocode** [comment]
I'd argue the current approach is more scalable. If a new output is added to the `sns_topic`, it will be much easier to support without having to update the module and release a new version.

**Comment by razorsedge** [comment]
But that argument only helps the module maintainers, not the module users.  It took me far too much time to figure out that a map was being returned and that I had to sort out which key I needed to use.  Completely unexpected compared to other Cloud Posse modules.  How will a less savvy user figure this out?

Will the maintainers be updating the README.md every time the AWS provider adds/removes an output to `aws_sns_topic`/`aws_sns_topic_subscriptions`?
